.github/workflows/build.yml

```diff
@@ -100,6 +104,9 @@ jobs:
           if [ "${{ matrix.name }}" = "dynamic" ]; then
             cargo test --release --features dynamic
           fi
+          if [ "${{ matrix.name }}" = "i686" ]; then
+            cargo test --target i686-unknown-linux-gnu --features dynamic
+          fi
```

Comment:
I think we should add also an appropriate condition to the `Build` stage. Right now it works (unlike the `dynamici` case which passes without executing anything) as the build is happening in the `Test` stage, but it makes the output a bit confusing.

I would probably suggest to make the `if` chain more explicit so the typos like this can get caugth with something like:
```
if X; then
 ...
elif Y; then
  ...
else
   fail unknown
fi
```

Reply:
build and test are frankly mostly redundant, they only help to distinguish at a glance if the issue is in the code or in the tests themselves, this i why for i686 I intentionally only build tests, as that covers all.